content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
 {{< code/copyable >}}
 {
   "require": {
-    "php": ">=7.1"
+    "php": ">=8.0"
   }
 }
 {{< /code/copyable >}}
 
 If your app has a `composer.lock` file, the buildpack will use
 the php version defined there.
 
+#### Deprecated: Using buildpack.yml
+
+Specifying the PHP version through `buildpack.yml` configuration will be
+deprecated in PHP language family buildpack v1.0.0.  To migrate from using
+`buildpack.yml` please set the `$BP_PHP_VERSION` environment variable.
+
+### Configure the PHP library directory
+
+Users can override the default libray code directory of `lib` for PHP by
+setting the `BP_PHP_LIB_DIR` environment variable at build-time.
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_LIB_DIR=some-lib-directory
+{{< /code/copyable >}}
+
+### Configure PHP with a custom `.ini` file
+If you'd like to configure custom `.ini` files in addition to the default
+`php.ini` provided by the [PHP Dist buildpack][bp/php-dist], users can provide
+their own configuration file in the application source directory under a
+directory named `.php.ini.d/`. Check out the reference docs about the
+[`PHP_INI_SCAN_DIR`]({{< ref "docs/reference/php-reference#php_ini_scan_dir"
+>}}) for more information about defaults.
```

Comment:
```suggestion
If you'd like to configure custom `.ini` files in addition to the default
`php.ini` provided by the [PHP Dist buildpack][bp/php-dist], you can provide
your own configuration file in the application source directory under a
directory named `.php.ini.d/`. Check out the reference docs about the
[`PHP_INI_SCAN_DIR`]({{< ref "docs/reference/php-reference#php_ini_scan_dir"
>}}) for more information about defaults.
```

Reply:
Looking at the PHP_INI_SCAN_DIR docs, it's not clear to me what the default value of the env var is – or whether that info is relevant to getting my custom `*.ini` file included in the build. Does a user need to set or update `PHP_INI_SCAN_DIR` to include a custom `*.ini`?